youtube_dl/extractor/youku.py

```diff
@@ -208,7 +234,8 @@ def retrieve_data(req_url, note):
             basic_data_url,
             'Downloading JSON metadata 1')
         data2 = retrieve_data(
-            'http://v.youku.com/player/getPlayList/VideoIDS/%s/Pf/4/ctype/12/ev/1' % video_id,
+            #'http://v.youku.com/player/getPlayList/VideoIDS/%s/Pf/4/ctype/12/ev/1' % video_id,
+            "http://play.youku.com/play/get.json?vid=%s&ct=12" % video_id,
```

Comment:
when I cloned the codes, data1 and data2 are already there. I try to not rewrite the code , instead fix in the original codes. So I keep data1 and data2.